From 27d1aa51dd819e8a2b20fa10b61ed7bd12a6645f Mon Sep 17 00:00:00 2001
From: Avery Pennarun <apenwarr@gmail.com>
Date: Tue, 1 Apr 2014 01:02:42 -0400
Subject: [PATCH] routeup: don't forget to flush stdout.

Otherwise, when stdout is to a pipe, it won't actually flush until many,
many routes have come up.
---
 src/routeup.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/routeup.c b/src/routeup.c
index 7cf04d7..0e1ce5f 100644
--- a/src/routeup.c
+++ b/src/routeup.c
@@ -134,7 +134,10 @@ main ()
   if (routeup_setup (&rtc))
     return 1;
   while (!routeup_once (&rtc, 0))
-    printf ("n\n");
+    {
+      printf ("n\n");
+      fflush (stdout);
+    }
   routeup_teardown (&rtc);
   return 0;
 }
-- 
1.9.1.423.g4596e3a

